src/components/picker/PickerDialog.android.js

```diff
@@ -17,6 +17,48 @@ class PickerDialog extends BaseComponent {
     onDone: PropTypes.func,
     onCancel: PropTypes.func,
     children: PropTypes.array,
+    /**
+     * Pass props for the WheelPicker (Android only)
+     */
+    wheelPickerProps: PropTypes.shape({
```

Comment:
If those are the WheelPicker props why are you re-defining them here?
Why not just use the WheelPicker.propTypes as we usually do?